app/javascript/src/components/Common/Sidebar/index.jsx

```diff
@@ -46,23 +48,26 @@ const Sidebar = () => {
   ];
 
   return (
-    <NeetoUISidebar
-      appName={APP_NAME}
-      changelogProps={{ id: "neetochangelog-trigger" }}
-      isCollapsed={isSidebarCollapsed}
-      navLinks={SIDENAV_LINKS}
-      organizationInfo={{
-        name: "Wheel",
-        subdomain: "bigbinary.com",
-      }}
-      profileInfo={{
-        name: `${user.first_name} ${user.last_name}`,
-        imageUrl: user.profile_image_path,
-        email: user.email,
-        bottomLinks,
-      }}
-      onCollapse={() => setIsSidebarCollapsed(!isSidebarCollapsed)}
-    />
+    <>
+      <NeetoUISidebar
+        appName={APP_NAME}
+        changelogProps={{ id: "neetochangelog-trigger" }}
+        isCollapsed={isSidebarCollapsed}
+        navLinks={SIDENAV_LINKS}
+        organizationInfo={{
+          name: "Wheel",
+          subdomain: "bigbinary.com",
+        }}
+        profileInfo={{
+          name: `${user.first_name} ${user.last_name}`,
+          imageUrl: user.profile_image_path,
+          email: user.email,
+          bottomLinks,
+        }}
+        onCollapse={() => setIsSidebarCollapsed(!isSidebarCollapsed)}
+      />
+      <NotesSidebar />
```

Comment:
Why do we have to keep the NotesSidebar in this component? So if we switch to the `Contact` page, then wouldn't we be seeing the NoteSidebar in that page too?

Reply:
Initially I kept component in index.js. I thought of changing this while working on the Contacts Page. Now I've changed it.